azure-devops/azext_devops/dev/common/vsts_git_url_info.py

```diff
@@ -91,7 +91,16 @@ def get_vsts_info(remote_url):
         else:
             uri = remote_url
         credentials = _get_credentials(uri)
-        return GitClient.get_vsts_info_by_remote_url(uri, credentials=credentials)
+        try:
+            return GitClient.get_vsts_info_by_remote_url(uri, credentials=credentials)
+        except Exception as ex:  # pylint: disable=broad-except
+            exceptionTypeName = type(ex).__name__
+            if exceptionTypeName == 'AzureDevOpsAuthenticationError':
+                logger.debug('Auto-detect from git url failed because of insufficient permissions.')
```

Comment:
Can we have a warning here, instead of just a debug log?

Reply:
Makes sense to me.. Since earlier this used to error out so warning will not be a drop in user experience and explicitly educate the user.

It is a PM call. @geverghe

Reply:
Making it warning after discussion with George.